include/sys/types.h

```diff
@@ -75,6 +75,7 @@ typedef struct pthread_mutexattr_t {
 typedef struct {
 	handle_t condh;
 	volatile int initialized;
+	clockid_t clock_id;
```

Comment:
I don't think we should add `clockid_t` here, without `condattr` it should just use some default value

Reply:
Ok nvm, I misunderstood this a little bit